static/src/js/util/addToast.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name addToast
+ * @param {React.Node} content
+ * @param {object} options Toast props for the notification
+ * @see https://github.com/jossmac/react-toast-notifications#toast-props
+ * @description Renders a toast notification.
+ */
+const addToast = (content, options) => {
```

Comment:
It feels strange that passing through the try/catch is a negative result (console.error)

First I'd remove the two `return` statements. Then change the if statement:
```
if (!add) {
  console.error('Add toast method not available.')
  return
}

add(content, options) 
```

Now when the code passes the try without an error it doesn't result in an error log

Reply:
Agreed @macrouch. I've cleaned up the code.